manuscript/090_How_to_start.md

```diff
@@ -284,14 +286,16 @@ ShouldAllowAccessToReportingWhenAskedForEitherAdministratorOrAuditor()
 }
 ```
 
-Start by invoking a�method when you have one
---------------------------------------------
+We still need to give this Statement a name, but given what we already know this is an easy task.
```

Comment:
I like how you moved this remark from parentheses to after the code, however, it also means we need to change the code snippet as in this snippet, the Statement is already named and after it is shown, there's a remark that we still need to give it a name.

I can think of three solutions:
1. The snippet after "at the final solution:" could be just the body of the method without signature and closing bracket and then it could be repeated in the final form after the remark about name.
2. Change the remark to something like "note that I already named this Statement - given what we already know, it was an easy task"

There may be other viable options, so feel free to suggest.